.travis.yml

```diff
@@ -48,6 +49,7 @@ before_deploy:
   - go get github.com/mattn/go-isatty
   - export build_file_name=wsk
   - export zip_file_name=OpenWhisk_CLI
+  - ./bin/godep restore -v
```

Comment:
For some reason two builds are happening. Why not just build once with travis.yml?????

Reply:
We want to distribute binaries that are actually tested. Rebuild the binaries with the wrong dependencies (not using Godeps) and not testing those binaries is not good.

Reply:
No need to add this line here. As you see in test_openwhisk.sh, we run make build, so the dependecy deps runs as well. All the godep dependencies are installed by  "go get -d -t ./..." .

Reply:
@dubeejw - good catch.  how many builds are now happening?